src/app/containers/Video/index.jsx

```diff
@@ -16,9 +17,32 @@ const VideoContainer = ({ blocks }) => {
     return null;
   }
 
+  const nestedModel = deepGet(['model', 'blocks', 0, 'model'], aresMediaBlock);
+  const kind = deepGet(['subType'], nestedModel);
```

Comment:
This uses `deepGet` as it will return a nicer value (`null` instead of `undefined`) if non-existent.